crdb/tx.go

```diff
@@ -21,6 +21,7 @@ package crdb
 import (
 	"database/sql"
 
+	"github.com/jinzhu/gorm"
```

Comment:
Wasn't sure how "official" `gorm` was and what the appetite for something like this would be, happy to refactor if this is on the line.  Would the introduction of more generic public `interface`s be preferable: one "Database" with which we could `Begin` a transaction and then the existing `Transaction` interface to `Execute`, `Commit`, and `Rollback`?